EasyLearn/app/views/events/index.html.erb

```diff
@@ -0,0 +1,132 @@
+<%= render "layouts/menu"%>
+
+<div class="container mt-3 col-10">
+<div class="d-flex justify-content-end m-3">
+  <button type="button" class="btn btn-success ml-3" data-toggle="modal" data-target="#addEvent">
+    Add
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-plus-square-fill" viewBox="0 0 16 16">
+      <path d="M2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2zm6.5 4.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3a.5.5 0 0 1 1 0z"></path>
+    </svg>
```

Comment:
What do you think about passing the SVG code to an external file?

Reply:
done